integration/cli/build.defaults.test.ts

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { beforeAll, describe, expect, it } from "@jest/globals";
+import * as path from "node:path";
+import * as fs from "node:fs/promises";
+import { Workdir } from "../helpers/workdir";
+import * as time from "../helpers/time";
+import * as pepr from "../helpers/pepr";
+
+const FILE = path.basename(__filename);
+const HERE = __dirname;
+
+describe("build", () => {
+  const workdir = new Workdir(`${FILE}`, `${HERE}/../testroot/cli`);
+
+  beforeAll(async () => {
+    await workdir.recreate();
+  });
+
+  describe("builds a module", () => {
+    const id = FILE.split(".").at(1);
+    const mod = `${workdir.path()}/${id}`;
```

Comment:
```suggestion
    const module = `${workdir.path()}/${id}`;
```

Avoiding abbreviations for short words is a personal style preference of mine

Reply:
A reasonable preference -- and one I agree with, generally -- when the variable is going to be exported / made part of a public API / used widely / live a long time, but... none of that really applies here.

_Not_ using reasonable shortenings just adds length to the code & makes it harder to scan, IMO, especially for the names of a local variables that are instantiated & die no more than 20 lines later.  But I don't feel that strongly about this case so sure, we can change it.  ¯\\_(ツ)_/¯

On this particular item though: using the name "module" would shadow a node built-in that I'd prefer to leave alone, so... I'll use "testModule" instead.  That work?

Reply:
`mod` -> `testModule` sounds good, thanks!